opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -23,13 +23,13 @@ describe('Raw streams tests', () => {
       expect(createEventsByTypes.label.length).toBe(15);
       expect(createEventsByTypes.identity.length).toBe(30);
       expect(createEventsByTypes.relationship.length).toBe(126);
-      expect(createEventsByTypes.indicator.length).toBe(31);
+      expect(createEventsByTypes.indicator.length).toBe(33);
```

Comment:
DecayRule integration test creates 2 indicators, so this is legit to me (same for others +2).